TF_Settings_Web/src/Components/Controls/TextEntry.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { ChangeEventHandler, MouseEventHandler, PointerEventHandler } from "react";
+
+import '../../Styles/Controls/TextEntry.css';
+
+interface TextEntryProps {
+    name: string,
+    value: number,
+    selected: boolean,
+    onChange: (ChangeEventHandler<HTMLInputElement>),
+    onClick: (MouseEventHandler<HTMLElement>),
+    onPointerDown: (PointerEventHandler<HTMLElement>)
+}
+
+export class TextEntry extends React.Component<TextEntryProps, {}> {
+    render() {
+        return(
+            <label onClick={this.props.onClick.bind(this)} onPointerDown={this.props.onPointerDown.bind(this)} className={"textEntryBackgroundLabel " + (this.props.selected ? "textEntryBackgroundLabelSelected" : "")}>
+                <p className="textEntryLabel">{this.props.name}</p>
+                <label className="textEntryContainer">
+                    <input type="number"
+                           step={0.01}
```

Comment:
Do we need the `step` property here? We're hiding the scroll buttons so there's no way to increment this item (I realise the same may be true of the number input in TextSlider in retrospect)